### DISCUSSION TITLE
Anyway to make a footpath go underground?

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22154] [Q&A] [answered]
I have a long train station platform and I need a way to go under it to reach another section of the park.   I can, of course,  build an elevated path to go over the platform but that takes a lot of cash to build all that.   A short underground tunnel would look a lot better.

**Comment by 733737** [comment, marked as answer]
use the build bridge/tunnel path tool while altering the land height with the land tool

**Reply by jcurbaniak** [reply]
Yes,  I just lowered the land under the station platform.   The platform stayed rigid and I could pass the pathway right under it.
